classification/support/view_cell_interactively.m

```diff
@@ -260,7 +260,7 @@ function show_other_cells(show)
         
         for m = 1:num_other_cells
             set(other_cell_handles(m,1), 'Visible', vis_val);
-            set(other_cell_handles(m,2), 'Visible', vis_val);
+%             set(other_cell_handles(m,2), 'Visible', vis_val);
```

Comment:
How about running the `text()` command here so we can still display the cell indices? If an average user does not use the neighbors functionality too often, the neighbors showing up late should not be a big issue.